tests/integration/targets/route53/tasks/main.yml

```diff
@@ -635,7 +635,304 @@
         - weighted_record is not failed
         - weighted_record is not changed
 
+#Test Geo Location - Continent Code
+  - name: Create a record with geo_location - continent_code (check_mode)
+    route53:
+      state: present
+      zone: '{{ zone_one }}'
+      record: 'geo-test-1.{{ zone_one }}'
+      identifier: "geohost1@www"
+      type: A
+      value: 127.0.0.1
+      ttl: 30
+      geo_location:
+        continent_code: NA
+    check_mode: true
+    register: create_geo_continent_check_mode
+  - assert:
+      that:
+        - create_geo_continent_check_mode is changed
+        - create_geo_continent_check_mode is not failed
+        - '"route53:ChangeResourceRecordSets" not in create_geo_continent_check_mode.resource_actions'
+
+  - name: Create a record with geo_location - continent_code
+    route53:
+      state: present
+      zone: '{{ zone_one }}'
+      record: 'geo-test-1.{{ zone_one }}'
+      identifier: "geohost1@www"
+      type: A
+      value: 127.0.0.1
+      ttl: 30
+      geo_location:
+        continent_code: NA
+    register: create_geo_continent
+  - assert:
+      that:
+        - create_geo_continent is changed
+        - create_geo_continent is not failed
+        - '"route53:ChangeResourceRecordSets" in create_geo_continent.resource_actions'
```

Comment:
is there any way we can validate that the record set's geo location was _actually_ changed? If not directly from the route53 module but from the route53_info module?

Reply:
Fixed.